gef.py

```diff
@@ -1269,6 +1261,18 @@ def sizeof(self) -> int:
     def addr(self) -> int:
         return int(self)
 
+    @property
+    def heap_start(self) -> int:
+        if self.ar_ptr - self.address < 0x60:
```

Comment:
please comment where special numbers like 0x60 come from

Reply:
you're right. i actually took a random value here which would fit the `heap_info` struct on all architectures and make sure that it is "close" to the start of the heap.
Currently, in glibc 2.37 and on x86_64, the `heap_info` struct is 48 bytes (0x30) big and I believe this would also fit this struct on all other architectures. What do you think - should I decrease the value to 0x30 and explain that it matches the biggest `heap_info` size currently throughout all architectures and glibc versions? Or should I leave it as is and explain that this is meant to be "future-proof" in case new fields will be added to the struct in upcoming versions?

Reply:
0x60 and explanation is fine with me.

Reply:
done